dm/syncer/error.go

```diff
@@ -61,6 +64,103 @@ func isDropColumnWithIndexError(err error) bool {
 			strings.Contains(mysqlErr.Message, "with tidb_enable_change_multi_schema is disable"))
 }
 
+// GetDDLStatusFromTiDB() retrieves the synchronizing status of DDL from TiDB
```

Comment:
```suggestion
// GetDDLStatusFromTiDB retrieves the synchronizing status of DDL from TiDB
```

Your IDE should display a warning for wrong format of function comment

Reply:
and maybe this function doesn't need to be public